backstage/packages/app/src/scaffolder/GithubTeamPicker/GithubTeamPicker.test.tsx

```diff
@@ -6,193 +6,196 @@ import { TestApiProvider } from '@backstage/test-utils';
 import { catalogApiRef } from '@backstage/plugin-catalog-react';
 import { Entity } from '@backstage/catalog-model';
 import {
-  ErrorApi,
```

Comment:
In the absence of standardized CI for this, could you put the exact command you ran to generate this diff in the commit message?

Reply:
the exact command? `⌘ + s` which saves to vsCode causing auto-format to run. I could have used `⌘+shift+p` to manually run prettier... which is the configured formatter...  I should probably just configure it to auto format on save  tbh

Reply:
lol, do you know how prettier is configured for your vscode? I just want to avoid the situation where two engineers are submitting competing reformats b/c one has a certain flag set

Reply:
this is actually defined in a dependency : @spotify/prettier-config